src/components/Auth/Registration/Registration.tsx

```diff
@@ -1,60 +1,74 @@
 import React from 'react';
-import { Box } from '@mui/material';
+import { Formik, Form } from 'formik';
 import { useTranslation } from 'react-i18next';
-import { useAppDispatch } from '../../../store/hooks';
-import { signin } from '../../../store/auth/authSlice';
-import RoutesList from '../../../navigation/routes';
-import Input from '../../../common/Input/Input';
-import { SubmitLink, SubmitButton } from '../Auth.style';
+import { CustomPaper, SubmitButton } from '../Auth.style';
+import validationSchema from './ValidationScheme';
+import FormikTextField from '../FormikTextField/FormikTextField';
 
 function Registration(): JSX.Element {
   const [t] = useTranslation();
 
-  const dispatch = useAppDispatch();
-  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
-  };
-
   return (
-    <Box component="form" onSubmit={handleSubmit} noValidate>
-      <Input
-        id="email"
-        label={t('auth.email')}
-        name="email"
-        autoComplete="email"
-      />
-      <Input
-        id="reemail"
-        label={t('auth.reemail')}
-        name="reemail"
-        autoComplete="reemail"
-      />
-      <Input
-        name="password"
-        label={t('auth.password')}
-        type="password"
-        id="password"
-        autoComplete="current-password"
-      />
-      <Input
-        name="repassword"
-        label={t('auth.repassword')}
-        type="password"
-        id="repassword"
-        autoComplete="current-password"
-      />
-      <SubmitLink to={RoutesList.Dashboard}>
-        <SubmitButton
-          type="submit"
-          fullWidth
-          variant="contained"
-          onClick={() => dispatch(signin())}
-        >
+    <Formik
+      initialValues={{
+        username: '',
+        age: '',
+        email: '',
+        password: '',
+        confirmPassword: '',
+        confirmEmail: '',
+      }}

```

Comment:
Please, add a new file `src/components/Auth/Registration/Registration.validation.ts` and put here schema and constants.

Also make this for LoginPage

Reply:
fixed